<launch>
    <param name='localizer/camera_params' value='$(find localizate_aruco)/config/c270_1280x720.yaml' />
    <param name='localizer/static_params' value='$(find localizate_aruco)/config/camera_pos.yaml' />
    <param name='localizer/GUI_enable' value='True' />
    <node name="cam"
            pkg="ocam_ros"
            type="ocam_ros_node"
            output="screen">
        <param name="device_path" value="/dev/video2" />
        <param name="width" value="1280" />
        <param name="height" value="720" />
        <param name="fps" value="60" />
        <param name="color" value="False" />
        <param name="image_topic" value="camera/image_raw" />
        <param name="show_image" value="False" />
    </node>
    
    <node name="localizer"
          pkg="localizate_aruco"
          type="main.py" 
          output="screen">

    </node>
</launch>